README.md

```diff
@@ -322,9 +306,7 @@ In `party.py`, there should be a function named `get_friends_unique_watched`. Th
 
 ### Wave 4
 
-1. There are four tests about a `get_available_recs` function
-
-Create a function named `get_available_recs`
+1. Create a function named `get_available_recs`. This function should...
 
 - takes one parameter: `user_data`
```

Comment:
I don't know if this is worth fixing with this PR, but I noticed there's some word agreement mismatching here and on the function descriptions below that the function descriptions above don't have.

```suggestion
- take one parameter: `user_data`
```

Reply:
Thanks! I'll take a closer look.

Reply:
changes takes to take